packages/typescript-plugin/src/index.ts

```diff
@@ -66,7 +89,11 @@ function init(modules: { typescript: typeof ts }) {
         }
     }
 
-    return { create, getExternalFiles };
+    function onConfigurationChanged(config: Configuration) {
```

Comment:
I saw this setting, too, in another plugin, but wasn't sure if it helps us. Does this mean we don't need to tell people to reload VS Code anymore?

Reply:
Yeah. My understanding is that tsserver would call this function when it receives the update config request from the typescript extension. The tslint plugin also opens a file system watcher to watch the tsconfig file. wondering if we should do that too. 

BTW if you have anything to add. Feel free to do so 😄.